app/views/audit.scala.html

```diff
@@ -782,10 +793,42 @@ <h1 class="text-center bold">@Messages("surface.problem")</h1>
         <div class="row">
             <div class="col-md-12">
                 <p>@Html(Messages("audit.tutorial.end.1"))</p>
+                <table style="width:100%;">
+                    <tbody><tr>
+                        <th style="text-align:center; width:50%">@Html(Messages("audit.tutorial.accessible"))</th>
+                        <th style="text-align:center; width:50%">@Html(Messages("audit.tutorial.inaccessible"))</th>
+                    </tr>
+                </tbody></table>
                 <p>
-                    <img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/ExamplePictures02.png")' id="onboarding-example-image-2"
+                    <img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/ExamplePictures02_top.png")' id="onboarding-example-image-2"
```

Comment:
You can also remove `ExamplePictures02.png` (and now that I'm looking at it, `ExamplePictures01.png` as well)